user/fonts-inter/template.py

```diff
@@ -0,0 +1,33 @@
+pkgname = "fonts-inter"
+pkgver = "4.0"
+pkgrel = 0
+pkgdesc = "Inter typeface family"
+maintainer = "beb <beb_@tutamail.com>"
+license = "OFL-1.1"
+url = "https://rsms.me/inter"
+source = f"https://github.com/rsms/inter/releases/download/v{pkgver}/Inter-{pkgver}.zip"
+sha256 = "ff970a5d4561a04f102a7cb781adbd6ac4e9b6c460914c7a101f15acb7f7d1a4"
+options = ["empty"]
+
+def do_install(self):
+    self.install_file("*.ttf", "usr/share/fonts/inter", glob=True)
+    self.install_file("extras/ttf/*.ttf", "usr/share/fonts/inter", glob=True)
+    self.install_file("extras/otf/*.otf", "usr/share/fonts/inter", glob=True)
+    self.install_license("LICENSE.txt")
```

Comment:
does OFL-1.1 need to be installed?

Reply:
it seems like the answer might be yes:

> 2) Original or Modified Versions of the Font Software may be bundled,
> redistributed and/or sold with any software, provided that each copy
> contains the above copyright notice and this license. These can be
> included either as stand-alone text files, human-readable headers or
> in the appropriate machine-readable metadata fields within text or
> binary files as long as those fields can be easily viewed by the user.

if so, we should lint this

Reply:
but anyway you don't need to do anything for this pr in particular, installing the license is correct here